packages/client/src/components/ui/input/input.module.scss

```
@@ -0,0 +1,26 @@
+@import '@/assets/variables.scss';
+
+.input-wrapper{
+
+  width: 100%;
+
+.label{
+  
+  display: block;
+  padding:0 0 2px 2px;
```

Не только js код нуждается в красивом форматирование :)
```suggestion
.input-wrapper {
  width: 100%;

  .label {
     display: block;
     padding: 0 0 2px 2px;
```

---

Согласен, все возлагаю надежды на prettier, нужно будет глянуть сможет ли он делать это за меня)